azdev/operations/linter/__init__.py

```diff
@@ -138,7 +138,7 @@ def run_linter(modules=None, rule_types=None, rules=None, ci_exclusions=None,
         command_loader, help_file_entries, modules=selected_mod_names, include_whl_extensions=include_whl_extensions)
 
     if not command_loader.command_table:
-        raise CLIError('No commands selected to check.')
+        logger.warning('No commands selected to check.')
```

Comment:
Will it work at line 155 if `not command_loader.command_table` is True? What's the value then?

Reply:
Yes. It still works for the following logic. It would exit with 0 as we expected.